test/installation_node/install from source with thread parameter

```diff
@@ -25,8 +25,11 @@ nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION || "'nvm run $NVM_T
 # Remove the stuff we're clobbering.
 [ -e ../../$NVM_TEST_VERSION ] && rm -R ../../$NVM_TEST_VERSION
 
-# Install from source with 2 make jobs (and swapped arg order)
-nvm install -j 2 -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+# Speed up test for non-CI environment which may have more CPU threads
+[ -z "${CI-}"] && nvm_get_make_jobs
```

Comment:
How much speed gain does this provide? I'm not sure how important it is to make tests fast only in local dev, compared to the cost of having tests do different things in CI.